test/test_cuda.py

```diff
@@ -673,7 +673,7 @@ def _check_default():
                 gcn_arch = str(
                     torch.cuda.get_device_properties(0).gcnArchName.split(":", 1)[0]
                 )
-                if gcn_arch in ["gfx90a", "gfx942", "gfx950"]:
+                if gcn_arch in ["gfx90a", "gfx942", "gfx950", "gfx1100", "gfx1101", "gfx1102", "gfx1150", "gfx1151", "gfx1200", "gfx1201"]:
                     self.assertTrue(default == torch._C._BlasBackend.Cublaslt)
```

Comment:
Should we also make this change on pytorch upstream?

https://github.com/pytorch/pytorch/blob/main/test/test_cuda.py

(I'm considering skipping this test case in TheRock until these test fixes land and I want to know which branches to skip the tests for... seems like all of them, at least until this PR and an upstream PR)